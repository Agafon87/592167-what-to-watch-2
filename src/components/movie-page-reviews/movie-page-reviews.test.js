import React from "react";
import renderer from "react-test-renderer";
import MoviePageReviews from "./movie-page-reviews";

it(`MoviePageReviews correctly renderers after test`, () => {
  const tree = renderer.create(<MoviePageReviews film={[]} />).toJSON();
  expect(tree).toMatchSnapshot();
});
