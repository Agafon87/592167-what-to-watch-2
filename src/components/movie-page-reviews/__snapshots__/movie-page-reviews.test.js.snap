// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`MoviePageReviews component renders correctly 1`] = `
<div
  className="movie-card__reviews movie-card__row"
>
  <div
    className="movie-card__reviews-col"
  />
  <div
    className="movie-card__reviews-col"
  />
</div>
`;
